lambdas/death-enrichment/src/test/java/uk/gov/di/data/lep/GroDeathEnrichmentTest.java

```diff
@@ -55,44 +57,40 @@ void constructionCallsCorrectInstantiation() {
     @Test
     void enrichGroDeathEventDataReturnsEnrichedData() throws JsonProcessingException {
         var sqsMessage = new SQSMessage();
-        sqsMessage.setBody("{\"sourceId\":\"123a1234-a12b-12a1-a123-123456789012\"}");
+        sqsMessage.setBody("A message body");
         var sqsEvent = new SQSEvent();
         sqsEvent.setRecords(List.of(sqsMessage));
 
-        when(objectMapper.readValue(sqsMessage.getBody(), GroDeathEventBaseData.class))
-            .thenReturn(new GroDeathEventBaseData(
-                "123a1234-a12b-12a1-a123-123456789012"
-            ));
+        when(objectMapper.readValue(sqsMessage.getBody(), GroJsonRecord.class))
+            .thenReturn(new GroJsonRecordBuilder().build());
```

Comment:
I don't _really_ care, but this feels a bit yucky. It would make more sense to me to have the enrichment (business logic) in a service class, which is called by the handler which acts like a controller and deals with details of the transport mechanism etc. You'd then be able to test it directly.